core/include/kernel/interrupt.h

```diff
@@ -57,7 +57,12 @@ struct itr_handler {
 	SLIST_ENTRY(itr_handler) link;
 };
 
-void itr_init(struct itr_chip *data);
+/*
+ * Initialise core interrupt controller driver
```

Comment:
I think `s/core/main/` would be more clear in case there is more than one interrupt controller.

Reply:
Agree. Are you ok I address your comment when squashing the fixup commits, instead of appending a messy fixup commit that affect all those platform files?

Note: I plan to update OP-TEE docs with this new labels.

Reply:
Sorry, I meesed-up. The many changed platform fiel are not impatced. I'll post a fixup commit.